mirascope/cohere/types.py

```diff
@@ -288,3 +291,41 @@ def tool_calls(self) -> Optional[list[ToolCall]]:
         if isinstance(self.chunk, StreamedChatResponse_ToolCallsGeneration):
             return self.chunk.tool_calls
         return None
+
+
+class CohereEmbeddingResponse(BaseEmbeddingResponse[SkipValidation[EmbedResponse]]):
+    """A convenience wrapper around the Cohere `EmbedResponse` response."""
+
+    embedding_type: Optional[
```

Comment:
seems confusing to have this here as well as the sequence of `embedding_types` in the params?

Reply:
dont have access to embedding_types in the response, i can pass the whole thing instead and grab the first one in the property instead.

Reply:
Oh I see. I think this is fine then since we're saying to access additional embeddings you can just access to original response and this is convenience for the first one.